helm/kyverno/charts/kyverno-policies/Chart.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v1
+name: kyverno-policies
+version: v2.1.1
+appVersion: v1.5.0
+icon: https://github.com/kyverno/kyverno/raw/main/img/logo.png
```

Let's please use a proper icon we own

```suggestion
icon: https://s.giantswarm.io/app-icons/kyverno/1/light.svg
```